backend/homework/4/src/main/java/shrishti/example/ExchangeArray/ArrayExchange.java

```
@@ -0,0 +1,21 @@
+package shrishti.example.ExchangeArray;
+import shrishti.example.Logger.logger;
+
+public class ArrayExchange {
+    public static <T> void exchangeELements(T[] arr,int pos1, int pos2){
+        if(arr==null || pos1<0 || pos1>=arr.length || pos2<0 || pos2>=arr.length){
+            logger.printLogger("Wrong input");
```

It would be better to throw an exception here.